ffs/scheduler/scheduler_storage.go

```diff
@@ -74,6 +75,9 @@ func (s *Scheduler) push(iid ffs.APIID, c cid.Cid, cfg ffs.StorageConfig, oldCid
 	if err := s.sjs.Enqueue(j); err != nil {
 		return ffs.EmptyJobID, fmt.Errorf("enqueuing job: %s", err)
 	}
+	if jid := s.sjs.GetExecutingJob(c); jid != nil {
+		s.l.Log(ctx, "Job %s is already being executed for the same data, this job will be queued until it finishes or is canceled.")
+	}
```

Comment:
If you push a new storageconfig for a Cid that already has an executing Job, we print an extra log line to explain that this job might take a while to get executed... since need the other job to finish (success, canceled, failed) before can be considered runnable.

This might help some people understand if they're in this situation.